app/presenters/nav_tab/link_collection.rb

```diff
@@ -27,15 +27,25 @@ def self.tab_methods
   end
 
   def admin
-    default + admin_only
+    use_button + admin_only
   end
 
   def customer
-    default + [orders, reservations, payment_sources, files]
+    default
   end
 
   def default
-    [home]
+    [orders, reservations, payment_sources, files]
+  end
+
+  def use_button
+    use
+  end
+
+  def home_button
+    if SettingsHelper.feature_on?(:use_manage) then use_button
```

Comment:
It's nitpicky, but I'm not really a fan of this style. We don't tend to use the `then` construct.

Can we either do:

```ruby
if SettingsHelper.feature_on?(:use_manage)
  use_button
else
  home
end
```

or

```ruby
SettingsHelper.feature_on?(:use_manage) ? use_button : home
```